lib/files.go

```diff
@@ -50,6 +50,7 @@ type localFile interface {
 	Content() io.ReadSeeker
 
 	Headers() map[string]string
+	ACL(storeACL string) string
```

Comment:
All of the rest looks good, but passing in a default value to a "getter" func seems ... clunky.

I understand that there is a global configuration (storeAcl) and a per route configuration, but it would make stuff easier to reason about if you just used the default if not set when creating the route config. With that a route will always have a value for `acl`.